### PR TITLE
solution 1.0

### PR DESCRIPTION
**Pull request by aliliolek**

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+// sorts an array
+function sortList(list) {
+  const arrEmpl = [...list].map(item => ({
+    ...item.dataset,
+    'name': item.innerText,
+  }));
+
+  arrEmpl.forEach(item => {
+    item.salary = item.salary.replace(/[^a-zа-яё0-9\s]/gi, '');
+    item.salary = Number(item.salary);
```
Comment (SanyaBratashchuk) [inline src/scripts/main.js:15]: Do not mutate data in data-attributes

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Comment by aliliolek** [comment]
https://aliliolek.github.io/js_task_parse_list_DOM/

File: src/scripts/main.js (hunk 1 above)
Comment (aliliolek) [reply src/scripts/main.js]: I thought I'm chainging the copy

File: src/index.html [hunk 2]
```diff
@@ -9,81 +9,48 @@
...
+       data-age="55">
         Rhona Davidson
       </li>
-      <li
-        data-position="Data Coordinator"
-        data-salary="$138,575"
-        data-age="64"
-      >
+      <li data-position="Data Coordinator" data-salary="$138,575"
+       data-age="64">
```
Comment (loralevitska) [inline src/index.html:37]: Why did you move the attributes, they were melted correctly, but now they are not. Put them back, please

File: src/index.html [hunk 3]
```diff
@@ -9,81 +9,48 @@
...
         Serge Baldwin
       </li>
-      <li
```
Comment (loralevitska) [inline src/index.html:61]: image

**Review by loralevitska — CHANGES_REQUESTED** [review]
good start)
let's make it better, put the html file back in place so it's not modified

File: src/index.html (hunk 3 above)
Comment (aliliolek) [reply src/index.html:61]: $ git push origin develop
remote: Permission to mate-academy/js_task_parse_list_DOM.git denied to aliliolek.
fatal: unable to access 'https://github.com/mate-academy/js_task_parse_list_DOM.git/': The requested URL returned error: 403
